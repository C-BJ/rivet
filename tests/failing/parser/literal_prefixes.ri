fn main() void {
    _ = b'A'; // OK
    _ = c'B'; // BAD

    _ = b"Byte string"; // OK
    _ = r"Raw string"; // OK

    _ = v"V string"; // BAD
}
