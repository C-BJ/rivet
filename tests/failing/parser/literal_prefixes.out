tests/failing/parser/literal_prefixes.ri:3:9: error: only `b` is recognized as a valid prefix for a character literal
    |
  3 |     _ = c'B'; // BAD
    |         ^
tests/failing/parser/literal_prefixes.ri:3:10: error: expected `;`, found character `B` 
    |
  3 |     _ = c'B'; // BAD
    |          ^
tests/failing/parser/literal_prefixes.ri:8:9: error: only `b` and `r` are recognized as valid prefixes for a string literal
    |
  8 |     _ = v"V string"; // BAD
    |         ^
tests/failing/parser/literal_prefixes.ri:8:10: error: expected `;`, found string `V string` 
    |
  8 |     _ = v"V string"; // BAD
    |          ^
rivetc: error: could not compile package `literal_prefixes`, aborting due to 4 previous errors.